src/librustc/ty/context.rs

```diff
@@ -1503,8 +1504,8 @@ impl<'gcx: 'tcx, 'tcx> GlobalCtxt<'gcx> {
 /// contain the TypeVariants key or if the address of the interned
 /// pointer differs. The latter case is possible if a primitive type,
 /// e.g. `()` or `u8`, was interned in a different context.
-pub trait Lift<'tcx> {
-    type Lifted: 'tcx;
+pub trait Lift<'tcx>: Debug {
+    type Lifted: Debug + 'tcx;
```

Comment:
AFAIK we usually import `fmt` and use `fmt::Debug`, is there any reason to do something different here.

Reply:
No particular reason, except that i've been gradually moving towards the more "defaut style" of "import capitalized things directly, invoke functions through modules". Happy to change.